assembl/static2/js/app/components/debate/common/topPostFormContainer.jsx

```diff
@@ -17,7 +17,9 @@ type TopPostFormContainerProps = {
   fillBodyLabelMsgId: string,
   bodyPlaceholderMsgId: string,
   postSuccessMsgId: string,
-  bodyMaxLength?: number
+  bodyMaxLength?: number,
+  draftable?: boolean,
```

Comment:
If I remember correctly, you don't need to use `?` if you have a default prop: https://flow.org/en/docs/react/components/#toc-using-default-props